docs/source/optim.rst

```diff
@@ -133,6 +133,15 @@ How to adjust Learning Rate
 rate based on the number of epochs. :class:`torch.optim.lr_scheduler.ReduceLROnPlateau`
 allows dynamic learning rate reducing based on some validation measurements.
 
+Learning rate scheduling should be applied after optimizer's update. The pattern is the following:
```

Comment:
Would it be helpful to mention here that this behavior has changed in the past?

Reply:
I agree. A `.. warning::` note or something similar would be helpful.

Reply:
I was thinking about that and then though that new users (who didn't know the previous pattern) are not concerned about the change between versions. But I agree, we can add a "warning" for those who knew the previous usage, saying that "There is a change between 1.0.1 and 1.1.0"... What do you think ?

Reply:
yeah, something like what we did for stft should work: https://pytorch.org/docs/master/torch.html#torch.stft

Reply:
@SsnL yes, this is a good example. I'll update the docs according to

Reply:
Thanks :)

Reply:
@SsnL I updated the text, please take a look if OK for you.